t5/data/utils.py

```diff
@@ -382,9 +382,10 @@ def __init__(self,
         `metric_fn(targets, predictions)` to use during evaluation. By default
         (None), an empty list will be used, resulting in no evaluation on this
         task.
-      postprocess_fn: function, a function that takes in decoded model outputs
-        (strings) and returns a string which is ready for evaluation using the
-        metric functions in `metric_fns`. Can be set to None as a no-op.
+      postprocess_fn: function (or list of functions) that (each) takes in
+        decoded model outputs (strings) and returns a string which is ready
+        for evaluation using the metric functions in `metric_fns`. Can be
+        set to None as a no-op.
```

Comment:
mention that if multiple functions are provided, they will be applied sequentially